pkg/restapi/operation/operations.go

```diff
@@ -340,6 +346,48 @@ func (c *Operation) onboardUser(id string) (*user.Profile, error) {
 	return userProfile, nil
 }
 
+func (c *Operation) handleBootstrapDataRequest(w http.ResponseWriter, r *http.Request) {
+	handle := r.URL.Query().Get("up")
+	if handle == "" {
+		handleAuthError(w, http.StatusBadRequest, "missing handle")
+
+		return
+	}
+
+	profile, err := user.NewStore(c.transientStore).Get(handle)
+	if errors.Is(err, storage.ErrValueNotFound) {
+		handleAuthError(w, http.StatusBadRequest, "invalid handle")
+
+		return
+	}
+
+	if err != nil {
+		handleAuthError(w, http.StatusInternalServerError,
+			fmt.Sprintf("failed to query transient store for handle : %s", err))
+
+		return
+	}
+
+	response, err := json.Marshal(&bootstrapData{
+		SDSURL:            c.bootstrapConfig.SDSURL,
+		SDSPrimaryVaultID: profile.SDSPrimaryVaultID,
+		KeyServerURL:      c.bootstrapConfig.KeyServerURL,
+		KeyStoreIDs:       profile.KeyStoreIDs,
+	})
+	if err != nil {
+		handleAuthError(w, http.StatusInternalServerError, fmt.Sprintf("failed to marshal bootstrap data : %s", err))
+
+		return
+	}
+
+	// TODO We should delete the handle from the transient store after writing the response,
+	//  but edge-core store API doesn't have a Delete() operation: https://github.com/trustbloc/edge-core/issues/45
```

Comment:
Let me know if not having the Delete() operation is blocking you and I can work on it

Reply:
@DRK3 it's not blocking for now but should be there if we are going to say this thing is "production-ready"